SonarRunner.Shim/Resources.resx

```diff
@@ -213,4 +213,13 @@
   <data name="ERR_ConflictingSonarProjectProperties" xml:space="preserve">
     <value>sonar-project.properties files are not understood by the SonarQube Scanner for MSBuild. Remove those files from the following folders: {0}</value>
   </data>
+  <data name="MSG_SarifFileNotFound" xml:space="preserve">
+    <value>No SARIF file found at {0}.</value>
```

Comment:
"SARIF" won't be meaningful to the end user. It would be better to say "Code Analysis ErrorLog file" instead.